streams/streams-scala/src/main/scala/org/apache/kafka/streams/scala/StreamsBuilder.scala

```diff
@@ -200,7 +200,7 @@ class StreamsBuilder(inner: StreamsBuilderJ = new StreamsBuilderJ) {
     consumed: Consumed[K, V],
     stateUpdateSupplier: org.apache.kafka.streams.processor.api.ProcessorSupplier[K, V, Void, Void]
   ): StreamsBuilderJ =
-    inner.addGlobalStore(storeBuilder, topic, consumed, stateUpdateSupplier)
+    inner.addGlobalStore(storeBuilder, topic, consumed, stateUpdateSupplier, true)
```

Comment:
Seem we would need to change `StreamsBuilder.scala`, too, and add an overload taking the new parameter? For this case, this KIP will need to cover this case.